lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -776,7 +776,9 @@ def _validate_single_linetime_pixelsize(self):
         line_times_err_msg = (
             ""
             if len(line_times) == 1
-            else (f"All source kymographs must have the same line times, got {sorted(line_times)} seconds.")
+            else (
+                f"All source kymographs must have the same line times, got {sorted(line_times)} seconds."
```

Comment:
sorting is no longer needed if you go with the `tuple` approach?

Reply:
actually you still need it because you're still making a `set` to make sure that you have a single value (even if it comes from 2 different kymos). So `._kymos` is `tuple` but what you're checking is still a (unordered) `set`

Reply:
Ah right, of course.